go/testutil/network.go

```
@@ -1 +1,62 @@
+		if err != nil {
+			return nil, err
+		}
+
+		listeners = append(listeners, l)
+		ports = append(ports, l.Addr().(*net.TCPAddr).Port)
+	}
+
+	return ports, nil
+}
+
+type freePorts struct {
```

golangci-lint (2.2.2)

type freePorts is unused

(unused)